### DISCUSSION TITLE
Where is admin demo?

### DISCUSSION THREAD

**Discussion by dausruddin** [mediacms-io/mediacms#401] [Q&A] [answered]
I wanted to see how admin panel look like, including the functionalities. Where do I get an access to the demo?

**Comment by Dabitch** [comment, marked as answer]
If by admin panel you mean the Django admin, this is what that looks like. 

<img width="1039" alt="djangoadmin" src="https://user-images.githubusercontent.com/5349307/151092648-3c6ece97-95a1-4932-ae55-f282db7e0455.png">

**Reply by dausruddin** [reply]
Thank you😁

**Reply by papayashake60** [reply]
Add Site?  Tried it ... but nothing happens

**Comment by mgogoulos** [comment]
> answer

Not sure what you mean here. But to clarify, there are two places to see functionality only admins can see:

a) the manage pages , which you get to see when you are logged in as admin and visit http://your_portal/manage/media, or http://your_portal/manage/users, or http://your_portal/manage/comments . These are 3 pages to edit media/users/comments, and are also available to editors/managers
b) the admin dashboard, where you get to see only if you are an admin, if you visit /admin

we could add screenshots to the documentation to point there. @papayashake60 if you mean 'Add Site'...does it mean you are already on the admin dashboard???
